src/javascript/app/pages/cashier/cashier.js

```diff
@@ -93,6 +93,8 @@ const Cashier = (() => {
         const has_upgrade         = upgrade_info.can_upgrade || upgrade_info.can_open_multi
                                     || can_change;
         const account_action_text = has_upgrade ? `<br />${localize('[_1]Manage your accounts[_2]', [`<a href=${Url.urlFor('user/accounts')}>`, '</a>'])}` : '';
+        const clients_country     = Client.get('residence') || State.getResponse('website_status.clients_country');
+        const change_text_for_im  = clients_country === 'im' ? 'time' : 'time or create an MT5 account';
```

Comment:
const is_iom_client = (Client.get('residence') || State.getResponse('website_status.clients_country')) === 'im';

Reply:
Applied the change, but did it a little different: 
is_iom_client = (Client.get('residence') === 'im' || State.getResponse('website_status.clients_country')) === 'im';

I believe both have to be specifically indicated with the === 'im', otherwise it will only read Client.get('residence') when that's true.